### PR TITLE
AAP-14300 Add verification steps for controller backup and restore

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR addresses the changes requested in https://issues.redhat.com/browse/AAP-14300.

Preview link (VPN required): http://file.rdu.redhat.com/~ddacosta/AAP14300/master.html

### CODE REVIEW

**Review by rooftopcellist — APPROVED** [review]

File: downstream/modules/platform/proc-aap-controller-backup.adoc [hunk 1]
```diff
@@ -41,3 +41,16 @@ $ df -h | grep "/var/lib/pgsql/data"
...
+. Log in to Red Hat *{OCP}*
+. Navigate to menu:Operators[Installed Operators].
+. Select the {OperatorPlatform} installed on your project namespace.
+. Select the *AutomationControllerBackup* tab.
+. Select the backup resource you want to verify.
+. Scroll to *Conditions* and check that the *Successful* status is `True`.
++
+[NOTE]
+====
+If *Successful* is `False`, the backup has failed. Check the automation controller operator logs for the error to fix the issue.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-aap-controller-backup.adoc:55]: should we use the attribute for automation controller here? Or is this acceptable with automation controller operator?

**Comment by jself-sudoku** [comment]
Just 2 questions about the use of AC attribute. Other than that, LGTM

File: downstream/modules/platform/proc-aap-controller-restore.adoc [hunk 2]
```diff
@@ -35,3 +35,17 @@ This should be different from the original deployment name.
...
+. Navigate to menu:Operators[Installed Operators].
+. Select the {OperatorPlatform} installed on your project namespace.
+. Select the *AutomationControllerRestore* tab.
+. Select the restore resource you want to verify.
+. Scroll to *Conditions* and check that the *Successful* status is `True`.
++
+[NOTE]
+====
+If *Successful* is `False`, the recovery has failed. Check the automation controller operator logs for the error to fix the issue.
+====
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-aap-controller-restore.adoc:51]: Same question as above about automation controller attribute

Comment (dcdacosta) [reply downstream/modules/platform/proc-aap-controller-restore.adoc:51]: Good catch, I've fixed this.

File: downstream/modules/platform/proc-aap-controller-backup.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-aap-controller-backup.adoc:55]: This is fixed now, thanks!

**Comment by jself-sudoku** [comment]
LGTM!

**Pull request merged**